### DISCUSSION TITLE
Unable to connect securely using DataGrip

### DISCUSSION THREAD

**Discussion by immjs** [planetscale/discussion#457] [Q&A]
Hello, I am trying to connect to my planetscale instance with DataGrip. However, even after ticking the `Use SSL` box, filling the CA file and setting Mode to Full Verification, it seems that the Connection Test is unsecure:
![](https://immjs.is-a-cool-femboy.xyz/62alGSFcS.png)

I assume the `Client certificate file` and `Client key file` are necessary, how could I gather them?

**Comment by orware** [comment]
Hello @immjs,

I haven't tried out DataGrip recently, but my guess here is that this might be a false negative with that `SSL: no` of the popup the connection test is presenting there in your screenshot.

Since our systems wouldn't allow the connection to succeed if it wasn't made securely, my guess here is that perhaps there isn't additional information presented to show that maybe the `TLS` protocol is being used instead of `SSL` here and that is the more likely reason to me why that would be showing up with `SSL: no` in the popup.

The settings you described, enabling `Use SSL`, providing the `CA file` location, and enabling `Full Verification` are all what we would recommend configuring for a connection to the database so from that perspective the settings are correct.

Providing a `Client certificate file` or `Client key file` isn't necessary for connecting securely to your PlanetScale database.
